### PR TITLE
Issue 64/projects section enhancements

### LINKED ISSUE #64: Projects Section Styling Enhacement
### Describe the Current Behavior/Feature:

- Cards should all be the same min-height
- Reduce padding below logos and text
- Change codePDX logo to new SVG with CODE PDX
- Make a clearer SVG of the PASS logo

---

### PR DESCRIPTION
**Pull request by andycwilliams**
## This PR:

Resolves #64

Creating a draft for now to gain feedback. Pretty minor changes at the moment. Before opening for review, I want to wait until #78 is complete to reduce potential conflicts.

---

## Screenshots (if applicable):

Desktop: 

Mobile:

---

## Future Steps/PRs Needed to Finish This Work (optional):

See if this satisfies all of the issue requirements, namely in regards to the logos.

### CODE REVIEW

**Review by Jared-Krajewski — APPROVED** [review]
made a push to center text, originally was just going to be for mobile but I think it looks better in all views in the cards. If you are ready this can be merged and the Image changes will come later on since I don't have time to try and make new images.

File: src/components/home/ProjectsBrief.jsx [hunk 1]
```diff
@@ -34,11 +35,10 @@ const renderProjectBriefCard = ({ title, description, logo }) => {
               mx: 'auto',
               objectFit: 'contain',
               maxWidth: '90%',
-              height: '70px',
-              mb: '15%'
+              height: '70px'
             }}
           />
-          <Typography variant="body1" sx={{ py: '1rem' }}>
+          <Typography variant="body1" textAlign="center" sx={{ py: { xs: '1rem', sm: '2rem' } }}>
```
Comment (xscottxbrownx) [inline src/components/home/ProjectsBrief.jsx:41]: line 42: `{description ?? null}`...

This is conditionally rendering the text inside the element, not the element itself. If the description is missing, then a `<p>` will still be put on the DOM just with nothing in it.

https://github.com/codeforpdx/codepdx_website/assets/71395931/042dfab9-2fa8-4ec4-9e50-3938d208aa76

Instead, what you want is to conditionally render the element like
`{description && (<Typography...>{description}</Typography>)}`

File: src/components/home/ProjectsBrief.jsx [hunk 2]
```diff
@@ -14,11 +14,12 @@ const cardStyle = {
...
+  m: { xs: 0, md: '0 25px 0 25px' },
+  p: { xs: 1, sm: 3 },
   background: 'rgba(217, 217, 217, 0.4)',
   borderRadius: '30px',
-  height: '100%'
+  height: { xs: 'auto', sm: '100%' },
+  minHeight: { xs: '0', sm: '500px' }
 };
 
 const renderProjectBriefCard = ({ title, description, logo }) => {
```
Comment (xscottxbrownx) [inline src/components/home/ProjectsBrief.jsx:25]: line 32: `image={logo ?? null}`...

similar to the `description` piece I mentioned.
This is conditionally giving or not giving the `img src` path. 

So, if there is no logo you get:
Screenshot 2023-12-29 at 9 42 04 PM

**Instead, what you want is to conditionally render the `<CardMedia>` (image element as a whole) like:**
`{logo && (<CardMedia...>)}`

---

Now, what happens if both `logo` and `description` don't exist? All you get is the `Learn More` section, so ideally you'd conditionally render a `<Grid>` only if either `logo` or `description` exist. Something like:

```js
const renderProjectBriefCard = ({ title, description, logo }) =>
  (logo || description) && (
    <Grid key={title} item xs={12} md={6} lg={4}>
      <Card sx={cardStyle}>
        <CardContent>
          {logo && (
            <CardMedia
              component="img"
              image={logo}
              alt={`${title} logo`}
              sx={{
                mx: 'auto',
                objectFit: 'contain',
                maxWidth: '90%',
                height: '70px'
              }}
            />
          )}
          {description && (
            <Typography variant="body1" textAlign="center" sx={{ py: { xs: '1rem', sm: '2rem' } }}>
              {description}
            </Typography>
          )}
        </CardContent>
        <CardActions>
          <Button href="/projects" size="large" color="primary">
            <strong>Learn More</strong>
          </Button>
        </CardActions>
      </Card>
    </Grid>
  );
  ```

**Review by xscottxbrownx — CHANGES_REQUESTED** [review]
conditional rendering issues

**Comment by xscottxbrownx** [comment]
I'll push a change for these issues (except conditionally rendering the whole grid item, as I believe nobody will put an entry in the projectsList array without at least one of these things.)

**Pull request merged**